backend/src/core/entities/chat.entity.ts

```diff
@@ -34,6 +40,9 @@ export class ChatEntity {
   @Column({ nullable: true })
   password: string;
 
+  @Column({ nullable: true })
+  createdByUser: number;
```

Comment:
Acho que aqui é um `UserEntity` ao invés de um number
E **talvez** fique mais idiomático chamar de `createdBy` 
```suggestion
  @Column({ nullable: true })
  createdBy: UserEntity;
```
(esse se aceitar teria que também mudar os outros lugares que tem `CreatedByUser`)

Reply:
Eu tentei trabalhar com UserEntity e criar a relacao no TypeORM. Mas fui infeliz tentando, essa abordagem foi a unica que funcionou sem problemas entaao subi assim mesmo kk. Se alguem quiser ajustaar a pr, feel free.

Reply:
TypeORM 😭